### DISCUSSION TITLE
Double the quantity of veins

### DISCUSSION THREAD

**Discussion by TrankiliumMC** [DerFrZocker/Ore-Control#32] [Q&A] [answered]
Hey, i would like, in one world, to double the quantity of veins of every ores.
I looked at the wiki, but I didn't quite understand what parameter I should change and what value I should set it to.
"Size" changes the amount of ores in a vein right? So to double the amount of veins (not the size of veins), do I have to change "Count" or "Rarity filter", or maybe another parameter? And what value should i set it to, will doubling the number double the amount of veins ?
Thank you in advance.

**Comment by DerFrZocker** [comment, marked as answer]
To double the amount of ores you need to double the amount of veins e.g. the `count` setting.
And yes `size` controls the amount of blocks a vein has.

For your use case the `rarity filter` can be ignored.
